src/Http/Offer/UpdateDescriptionRequestHandler.php

```diff
@@ -47,26 +50,35 @@ public function handle(ServerRequestInterface $request): ResponseInterface
             new DenormalizingRequestBodyParser(new DescriptionDenormalizer(), Description::class)
         );
 
-        $request = $requestBodyParser->parse($request);
+        try {
+            $request = $requestBodyParser->parse($request);
 
-        /** @var Description $description */
-        $description = $request->getParsedBody();
+            /** @var Description $description */
+            $description = $request->getParsedBody();
 
-        if ($offerType->sameAs(OfferType::event())) {
-            $updateDescription = new EventUpdateDescription(
-                $offerId,
-                $language,
-                $description
-            );
-        } else {
-            $updateDescription = new PlaceUpdateDescription(
+            if ($offerType->sameAs(OfferType::event())) {
```

Comment:
Can we make either a seperate function or class helper that expects any type of UpdateDescription and just returns the correct type of Command?

Reply:
Would be useful for example to have an OfferUpdateDescription class with a static method "buildCommand()" that returns the correct object? 
If you think that is overkill I am also happy with just a function in this handler.

Reply:
The approach described here should be used https://jira.publiq.be/browse/III-3834. This is not inside the scope of this refactor of value objects and that is why I didn't change it.

Reply:
I will approve then